### PR TITLE
Added: Documentation on POS Order refresh failure

### PR DESCRIPTION
**Pull request by adayush17**
Changelog:

Added the document on POS Order Refresh Failure, Additionally added the images.

### CODE REVIEW

File: everything/sop/pos-order-refresh-failure.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+description: >-
+This document provides steps to resolve the 'POS Order Refresh Failure' issue in Hotwax Commerce OMS.
```
Comment (Amir-HC) [inline everything/sop/pos-order-refresh-failure.md]: Use double quotes for POS Order Refresh Failure

File: everything/sop/pos-order-refresh-failure.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+# POS Order Refresh Failure in Hotwax Commerce OMS
+
+
+POS orders are downloaded as fulfilled in HotWax Commerce from Shopify. However, there are instances when the order is downloaded in OMS before it's marked fulfilled in Shopify. In such cases, retailers have to refresh the order to ensure fulfilled orders are downloaded while the unfulfilled ones get canceled.
+
+However, sometimes the orders are not refreshed due to missing shipping addresses. The error message received when hitting the “Refresh Order” button is "Could not complete the createOrderContactMech process: The following required parameter is missing: [createOrderContactMech.contactMechId]".
+
+In such cases, such orders need to be manually removed, and the updated order needs to be imported again.
```
Comment (Amir-HC) [inline everything/sop/pos-order-refresh-failure.md]: i think this is the same as I had written, need to ChatGPT it once

File: everything/sop/pos-order-refresh-failure.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Here are the steps to troubleshoot these errors:
+
+## Troubleshooting Steps
+
+### Step 1: Cancel the Order
+1. **Log in to OMS**: Use your username and password to log in to Hotwax Commerce OMS.
+2. **Navigate to Sales Orders**: Click the hamburger navbar icon if the left slider is not visible. Go to "Sales Orders" under "Order Management" in the left slider.
+3. **Identify the Order**: Find the order exhibiting the error. Open the order by clicking on its ID.
+4. **Cancel the Order**: On the order page, click the "Cancel" button at the top.
```
Comment (Amir-HC) [inline everything/sop/pos-order-refresh-failure.md]: use backticks instead of double quotes

Comment (Amir-HC) [reply everything/sop/pos-order-refresh-failure.md]: in all the places where backticks need to be added

**Review by Amir-HC — COMMENTED** [review]
general review

read documentation guidelines and add quotations accordingly

File: everything/sop/pos-order-refresh-failure.md (hunk 2 above)
Comment (adayush17) [reply everything/sop/pos-order-refresh-failure.md]: Okay sir

File: everything/sop/pos-order-refresh-failure.md (hunk 3 above)
Comment (adayush17) [reply everything/sop/pos-order-refresh-failure.md]: got it

**Pull request merged**